### DISCUSSION TITLE
VoiceChannel topic edit?

### DISCUSSION THREAD

**Discussion by moa12gg** [discordjs/discord.js#10169] [Q&A] [answered]
How do I change the topic of the voice channel? (v13)

I tried the following:
`client.channels.cache.get('11598340623541xxxx').edit({topic:'test1'}).then(console.log).catch(console.error);`
But this doesn't work.
-------DiscordAPIError: Invalid Form Body
-------topic: Channel topic contains at least one word that is not allowed.

another attempt:
`client.channels.cache.get('11598340623541xxxx').setToic("test").then(console.log).catch(console.error);`
But this doesn't work.
------- TypeError: client.channels.cache.get(...).setTopic is not a function

![20240306_214451](https://github.com/discordjs/discord.js/assets/92944493/1e2623e6-cbe3-4438-84a2-a240267e7697)

Is there a way?
Sorry I used Google Translator.

**Comment by almostSouji** [comment, marked as answer]
This does not fall under "topic" but "voice-status"
It is currently not available in discord.js (either version), while discord decides on documenting it, see https://github.com/discord/discord-api-docs/pull/6398
